<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizador de Codificação de Linha</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Input:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>Binary</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Hexadecimal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Text</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Partial Visualization</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Horizontal Position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollBar" name="horizontalScrollBar">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="mainLayout"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1364</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_As_Image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_LCV"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Input"/>
    <addaction name="actionShow_Clock"/>
    <addaction name="actionShow_Vertical_Grid"/>
    <addaction name="actionShow_Horizontal_Grid"/>
    <addaction name="actionShow_Top_Spine"/>
    <addaction name="actionShow_Right_Spine"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAjuda">
   <property name="text">
    <string>&amp;Ajuda</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>&amp;Sobre</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSave_As_Image">
   <property name="text">
    <string>Save As Image...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionAbout_LCV">
   <property name="text">
    <string>About LCV</string>
   </property>
  </action>
  <action name="actionShow_Input">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Input</string>
   </property>
  </action>
  <action name="actionShow_Clock">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Clock</string>
   </property>
  </action>
  <action name="actionShow_Vertical_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Vertical Grid</string>
   </property>
  </action>
  <action name="actionShow_Horizontal_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Horizontal Grid</string>
   </property>
  </action>
  <action name="actionShow_Top_Spine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Top Spine</string>
   </property>
  </action>
  <action name="actionShow_Right_Spine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Right Spine</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
